src/App.js

```diff
@@ -21,22 +28,47 @@ class App extends Component {
           })
         })
         .catch(error => console.log(error))
-        console.log(this.state.countryData)
+
+        console.log('fetched')
+    }
```

Comment:
Great work on these methods! This allows us to have our data on page load, as we are calling both methods to grap our data in our 'componentDidMount()' methods.